Trippy/Models/RouteLeg.m

```diff
@@ -0,0 +1,37 @@
+//
+//  RouteLeg.m
+//  Trippy
+//
+//  Created by Catherine Lu on 7/11/22.
+//
+
+#import "RouteLeg.h"
+
+@implementation RouteLeg
+
+- (instancetype) initWithDictionary:(NSDictionary *)dict{
+    self = [super init];
+    
+    if (self) {
+        self.distanceText = dict[@"distance"][@"text"];
```

Comment:
What is the distance text here? Do you need to store it? Is it possible to build from the val?

Reply:
Text states distance in miles, whereas the val is in...meters I think? I could probably convert between the two, will update!